### ISSUE TITLE
Comments for "Braun Cruzer3 battery replacement"

### ISSUE THREAD

**Issue by spbnick** [spbnick/spbnick.github.io#5]
Please leave your comments for article "[Braun Cruzer3 battery replacement](http://spbnick.github.io/2016/06/24/Braun-Cruzer3-battery-replacement.html)" here.

**Comment by tabourider** [comment]
A very helpful no-nonsense tutorial - practical, well illustrated and mercifully free from nerdy self advertisement unlike most utube vids - thank you. I look forward to getting my new batteries!

**Comment by spbnick** [comment]
Thanks for the kind feedback, @tabourider. Good luck with your repair! Please write if you find any problem with my instructions.

**Comment by tabourider** [comment]
Hi Nikolai

Very kind of you to respond! I probably bought my Cruzer about the same
time as you got yours as it's down to three shaves if I'm quick. Your
mention of the security tabs breaking made me wonder where from exactly? Do
they shake out easily so that they don't foul any of the mechanicals?

On 9 July 2016 at 17:32, Nikolai Kondrashov notifications@github.com
wrote:

> Thanks for the kind feedback, @tabourider https://github.com/tabourider.
> Good luck with your repair! Please write if you find any problem with my
> instructions.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/spbnick/spbnick.github.io/issues/5#issuecomment-231542857,
> or mute the thread
> https://github.com/notifications/unsubscribe/ATbNqRLNCAoX2pPNkUNjRPaPX7d02D7Uks5qT80DgaJpZM4I9zUW
> .

## 

Kindest regards

Dave

**Comment by spbnick** [comment]
Well, I haven't found any. I also didn't hear them rattling about. Perhaps they are small and are left hanging on a piece of bent plastic.

**Comment by tabourider** [comment]
Thanks for your quick reply and reassurance, Nikolai. Have a great day!

On 10 July 2016 at 13:59, Nikolai Kondrashov notifications@github.com
wrote:

> Well, I haven't found any. I also didn't hear them rattling about. Perhaps
> they are small and are left hanging on a piece of bent plastic.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/spbnick/spbnick.github.io/issues/5#issuecomment-231587793,
> or mute the thread
> https://github.com/notifications/unsubscribe/ATbNqSopV7gORJj6OoVTWsIbk3Ro4YFvks5qUOykgaJpZM4I9zUW
> .

## 

Kindest regards

Dave

**Comment by usolomer** [comment]
Thanks for the tutorial. Seems this is the time when the batteries for that model die. Just replaced mine with the help of your tutorial. Didn't expect to find a tutorial at all.

Some note thought:
I hadn't realized that the device switches itself off internally as soon as you remove the battery case. So even if you reassemble everything immediatly after, without replacing anything, you can't switch it on again. Only after you placed the device into the loading craddle for a few seconds it will turn itself on again and work as usual. Bear that in mind, when replacing your batteries.

**Comment by spbnick** [comment]
@usolomer Yeah, that's what I tried to describe at the end. I was puzzled at first too. Glad to hear my instructions helped!

**Comment by maschibl** [comment]
thanks for the nice description! just successfully replaced my batteries.

**Comment by spbnick** [comment]
Hey, @maschibl, glad to hear it worked for you and we saved another device from the landfill :).

**Comment by gpipito** [comment]
Hi @spbnick , many thanks for your precious tutorial. Since I'm not good at soldering but want to try out your procedure anyway, did you use any soldering wire for the copper parts?

**Comment by spbnick** [comment]
@gpipito, you have to use solder for soldering, and normally it comes as
soldering wire. Soldering is not hard. Make sure you get rosin-core
soldering wire, not more than maybe 1mm thick, watch a couple tutorials on
youtube, practice a bit, and you should be good to go!

**Comment by warren-livingstone** [comment]
These instructions are excellent.  I have the Braun CruZer5, but it is pretty much the same shaver.  I did find that you can purchase the correct batteries from a site in the US....http://www.electricshaverstore.com/m/4/braun_2.htm.  the batteries are $7.95 USD each, so it would save the soldering steps in these instructions.  I have ordered the batteries and will finish putting it back together then.  Thanks for the help.

**Comment by warren-livingstone** [comment]
...part 2; batteries arrived and were identical to the dead ones.  Glad you had the comment about it not working until you plug it in and disconnect...I was concerned :).  Thanks again for the great tutorial.

**Comment by spbnick** [comment]
@warren-livingstone, glad to hear you found the article useful, and it's great that you could find the batteries! I'm sure others will find it easier to use them rather than mess around with soldering.

**Comment by wolfiex** [comment]
Great article - In my case, I had not used the device for 6 months and had left it on empty. Removing the batteries, and putting them into a battery charger (with the aid of a couple of magnets) and then replacing them overrode the charging circuitry and made them usable once again. I just had to make sure they were the correct orientation, for which the photos helped!

**Comment by orborde** [comment]
1. In mine, the batteries were soldered to the contacts. I was able to get them out by holding the soldering iron to the contact, melting the solder, and simultaneously having a friend gently pull the battery out using pliers.

2. Instead of soldering in contact points like you did, I bought special replacement batteries that had the pins already built-in. I bought these: https://shaveroutlet.com/braun-aaa-nimh-replacement-battery-with-snap-in-pins.html . They seem to work so far!

**Comment by tsuiwilliam** [comment]
awesome tutorial, gave me the confidence to open mine up. between this tutorial and some korean dude's voiceless YT https://www.youtube.com/watch?v=_1UKTbYFyfI, i just dropped in a couple old eneloops found at home and bent the contacts like the video... no soldering needed =) THANKS!!! IT'S ALIVE AGAIN!!!

**Comment by andrew-gabriel** [comment]
Thanks for this. I just replaced the batteries in my 2876/5733 cruZer3.
The case slid apart easily once the rear screw was removed - there were no extra locking tabs.
The forked battery tabs had something soldered bridging the tips of the fork, so they had to be bent back quite some distance to enable the pins to pop out. After that, I bent the tabs inwards to better grip standard AAA NiMH batteries.

Also, with the age of this shaver, I took the opportunity to wipe down the rubberized cover which was starting to go sticky, with isopropyl alcohol, which has cleaned it up nicely.
